daemon/graphdriver/vfs/driver.go

```diff
@@ -32,18 +35,22 @@ func (d *Driver) String() string {
 	return "vfs"
 }
 
+// Status of the driver should be returned, a nil array is returned for VFS driver.
 func (d *Driver) Status() [][2]string {
 	return nil
 }
 
+// GetMetadata returns a nil map and error for VFS driver.
 func (d *Driver) GetMetadata(id string) (map[string]string, error) {
 	return nil, nil
 }
 
+// Cleanup returns nil for the error for the VFS driver.
```

Comment:
Cleanup is used to implement `graphdriver.ProtoDriver`. There is no cleanup required for this driver.